test/unit/component/parser/xml/BpmnXmlParser.exclusiveGateway.test.ts

```diff
@@ -66,11 +66,11 @@ describe('parse bpmn as xml for exclusive gateawy', () => {
     verifyBounds(shapes[1], 852, 282, 46, 45);
   });
 
-  it('bpmn with multiple processes, ensure exclusive gateawy are present', () => {
+  it('bpmn with multiple processes, ensure exclusive gateway are present', () => {
```

Comment:
Is it possible to have the same test for task, user task, start event, end event also ?

Reply:
If your question is about `decoding xml entities`, we could but I think this is useless as we are always doing the same processing for attribute retrieval
If you think this is important, let me know and decide how we can manage it in a clever way 😸

Reply:
I don't know if it's very important, but as we verified if we can decoding all xml objects, if I think to keep coherence, we need to verify if we can decoding all as the same way.

Reply:
Let's talk about this directly

Reply:
Decisions taken
- leave as it is for now: tests ensure that xml entities in process elements (doing this only with exclusive gateway is enough) and participants are correctly managed
- in the future, we will change the way we do xml parsing testing, see #156